warehouse/models/mart/transit_database/dim_mobility_mart_providers.sql

```diff
@@ -0,0 +1,160 @@
+{{ config(materialized='table') }}
+
+-- An Organization (i.e. dim_organizations) can provide multiple different
+-- Services (i.e. dim_services). Each Service will have its own funding sources,
+-- GTFS feeds, and target counties.
+
+WITH organizations AS (
+   SELECT * FROM {{ ref('dim_organizations') }}
+   WHERE _is_current
+),
+
+services AS (
+    SELECT * FROM {{ ref('dim_services') }}
+    WHERE _is_current
+),
+
+county_geogs AS (
+    SELECT * FROM {{ ref('dim_county_geography') }}
```

Comment:
It may be worth adding the `_is_current` filter on all these just to future proof